mediaflow_proxy/extractors/vixcloud.py

```
@@ -0,0 +1,65 @@
+        parsed_url = urlparse(iframe)
+        query_params = parse_qs(parsed_url.query)
+        response = await self._make_request(iframe, headers={"x-inertia": "true", "x-inertia-version": version})
+
+        if response.status_code != 200:
+            raise ExtractorError("Failed to extract URL components, Invalid Request")
+        soup = BeautifulSoup(response.text, "lxml", parse_only=SoupStrainer("body"))
+        if soup:
+            script = soup.find("body").find("script").text
+            token = re.search(r"'token':\s*'(\w+)'", script).group(1)
+            expires = re.search(r"'expires':\s*'(\d+)'", script).group(1)
+            quality = re.search(r'"quality":(\d+)', script).group(1)
```

Ruff (0.8.2)

Local variable `quality` is assigned to but never used

Remove assignment to unused variable `quality`

(F841)